src/components/BeaconSummaryPanel.tsx

```diff
@@ -48,20 +42,67 @@ export const BeaconSummaryPanel: FunctionComponent<IBeaconSummaryProps> = ({
         setBeacon(beacon);
         setState(DataPanelStates.Viewing);
       } catch (error) {
-        console.log(error);
+        console.error(error);
         setState(DataPanelStates.Error);
       }
     };
 
     fetchBeacon(beaconId);
   }, []); // eslint-disable-line
 
+  const fields = [
+    {
+      key: "Manufacturer",
+      value: beacon?.manufacturer,
+    },
+    {
+      key: "Model",
+      value: beacon?.model,
+    },
+    {
+      key: "Beacon type",
+      value: beacon?.type,
+    },
+    {
+      key: "Protocol code",
+      value: beacon?.protocolCode,
+    },
+    {
+      key: "Serial number",
+      value: beacon?.manufacturerSerialNumber,
+    },
+    {
+      key: "CHK code",
+      value: beacon?.chkCode,
+    },
+    {
+      key: "Battery expiry date",
+      value: beacon?.batteryExpiryDate,
+    },
+    {
+      key: "Last serviced date",
+      value: beacon?.lastServicedDate,
+    },
+    {
+      key: "Owner(s)",
+      value: formatOwners(beacon?.owners || []),
+    },
+    {
+      key: "Emergency contacts",
+      value: formatEmergencyContacts(beacon?.emergencyContacts || []),
+    },
+    {
+      key: "Registered uses",
+      value: formatUses(beacon?.uses || []),
+    },
+  ];
+
```

Comment:
This code maps the `IBeacon` to tabulated form for display/interaction in the `Panel`.  I'm leaving this here _for now_, but I can smell a `Tabulator` class emerging that does this transformation in between the component and the gateway object.

Reply:
NB - I initially thought about abstracting this further, to programmatically turn any of the entities (`IBeacon`, `IUse` etc.) into an array of key/value pairs, but the nested data structures in `owners`, `emergencyContacts` and `uses` scuppered that.  Each requires its own particular `format` function.